extensions/ftp/pom.xml

```diff
@@ -29,11 +29,38 @@
             <groupId>commons-net</groupId>
             <artifactId>commons-net</artifactId>
         </dependency>
+
+        <dependency>
```

Comment:
check with Kraan, but I'm under the impression that the `java` and `javax` ones are going to be runtime only